lib/ProMotion/table/cell/table_view_cell_module.rb

```diff
@@ -163,9 +163,7 @@ def map_content_mode_symbol(symbol)
 
     def map_selection_style_symbol(symbol)
       {
-        none:     UITableViewCellSelectionStyleNone,
-        blue:     UITableViewCellSelectionStyleBlue,
```

Comment:
We probably don't want to remove these keys yet (without doing a major version bump), as some people may be referencing them in their codebase.